features/google/src/components/importExplanationExpandable/importExplanationExpandable.css

```diff
@@ -31,14 +31,14 @@
     align-items: center;
     padding: calc(var(--gdu) * 6);
     box-sizing: border-box;
+    font-family: var(--jost-medium);
+    font-size: var(--font-size-xl);
 }
```

Comment:
I followed styles in Figma: https://www.figma.com/file/xpeZC2jl9JIDqt1kiec5US/MAIN-pod-prototype?node-id=10955%3A7345 , so this would be the correct fonts. However, I found this other ticket: https://jira.polypoly.eu/browse/PROD4POD-1644 requesting both facebook and google feature to have the same "expansion panels". Since we are in feature freeze, I am not sure about what decision to make here. Should I leave them as they were ("wrong" fonts, but same styles in both features) and fix them later?

Reply:
I think you can safely change the fonts.